docs/source/quickstart_drilldown.rst

```diff
@@ -0,0 +1,144 @@
+:orphan:
+
+.. _quickstart_drilldown:
+
+Quickstart additional material: Options and troubleshooting
+===========================================================
+
+
+.. 
+    Hmm... At the moment, I have this as a single separate document.
+    Other options would be to inline this below the 'next steps' section of the quickstart doc itself. That would probably be a net gain for SEO, at the expense of a longer doc. 
+
+    The other option would be to have each section here as a separate doc, and link to them from the quickstart doc. Would that be a gain for SEO (doc is better than section?)
+
+
+.. _quickstart_drilldown_install:
+
+Customizing and troubleshooting MLflow installation
+---------------------------------------------------
+
+tk
+- Java exists
+- R exists
+- mlflow-skinny
+- mlflow[extras]
+- Mac default python
+- environment managers
+- conda channel exists?
+tk
+
+.. _quickstart_drilldown_autolog:
+
+Autolog options
+---------------
+
+tk
+
+.. _quickstart_drilldown_tracking_api:
+
+Effectively using the MLflow Tracking API
+-----------------------------------------
+
+tk
+
+.. _quickstart_drilldown_tracking_ui:
+
+Navigating the MLflow Tracking UI
+---------------------------------
+
+tk
+
```

Comment:
examples of how to use the search functionality :) There are a lot of issues filed with users being confused by the syntax / utility of it

Reply:
TODO